docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing to "project"
+
+The main purpose of this project is "objective"
+
+## Summary
+
+* [Code of Conduct](#code-of-conduct)
+* [Issues](#Issues)
+* [Pull Requests](#pull-requests)
+* [Git workflow](#git-workflow)
+* [Join the Project](#join-us)
+
+## Code of Conduct
+
+By participating and contributing to this project, you agree to uphold these rules:
+
+- Give credit where credit is due
+- Be respectful to all project participants
+- Follow the `Code of Conduct` from [contributor-covenant.org](https://www.contributor-covenant.org/version/1/4/code-of-conduct/)
+
+
+### Issues
+
+**Purpose**:
+- Documenting critical issues, bugs, and other concerns
+- Tracking topics/ questions that need to be researched
+- For proposing new features, suggestions, or enhancements
+
+### Pull Requests (PR)
+
+**Guidelines**
+- PRs must be associated with an issue
+- PRs should only address ONE project related concern
+- If PR is rejected, code reviewer must provide reason for rejection
+- PRs must be reviewed and tested* (*unless the code is small enough to not require testing*)
+  - To test the code, code reviewer must pull the PR, run/test the application, and run unit tests (if any)
```

Comment:
We will probably need to setup a testing framework, but we can keep this in for now.